### DISCUSSION TITLE
Compile from source failing in make, linux

### DISCUSSION THREAD

**Discussion by tckosvic** [Stellarium/stellarium#3422] [General]
Compilation stops:  referencing line 156 of Makefile. See below:

`[ 78%] Linking CXX static library libTelescopeControl_Lx200.a
[ 78%] Built target TelescopeControl_Lx200
[ 78%] Linking CXX static library libQXlsxQt5.a
[ 78%] Built target QXlsx
make: *** [Makefile:156: all] Error 2
(base) tom@mydesktop:~/stellarium/build/unix> `

Using most recent source code download; not from git.  Any suggestions?

thanks,    tom kosvic

**Comment by ilkant** [comment]
Have you done cmake in build directory (for example Linux: build/unix)?
